Task2/aga/TicTacToe/TicTacToe/TheGame.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<Window x:Class="TicTacToe.TheGame"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:TicTacToe"
+        mc:Ignorable="d"
+        Title="The Game" Height="270" Width="300">
+    <Grid Margin="10">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="50"/>
+            <RowDefinition Height="10"/>
+            <RowDefinition Height="50"/>
+            <RowDefinition Height="10"/>
+            <RowDefinition Height="50"/>
+            <RowDefinition Height="10"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="50"/>
+            <ColumnDefinition Width="10"/>
+            <ColumnDefinition Width="50"/>
+            <ColumnDefinition Width="10"/>
+            <ColumnDefinition Width="50"/>
+            <ColumnDefinition Width="10"/>
+            <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+        <Button Width="50" Height="50" Name="A1" Click="A1_Click"/>
+        <Button Width="50" Height="50" Name="A2" Grid.Row="2" Click="A2_Click"/>
+        <Button Width="50" Height="50" Name="A3" Grid.Row="4" Click="A3_Click"/>
+        <Button Width="50" Height="50" Name="B1" Grid.Column="2" Click="B1_Click"/>
+        <Button Width="50" Height="50" Name="B2" Grid.Row="2" Grid.Column="2" Click="B2_Click"/>
+        <Button Width="50" Height="50" Name="B3" Grid.Row="4" Grid.Column="2" Click="B3_Click"/>
+        <Button Width="50" Height="50" Name="C1" Grid.Column="4" Click="C1_Click"/>
+        <Button Width="50" Height="50" Name="C2" Grid.Row="2" Grid.Column="4" Click="C2_Click"/>
+        <Button Width="50" Height="50" Name="C3" Grid.Row="4" Grid.Column="4" Click="C3_Click"/>
+        <Label x:Name="Turn" Content="Turn" Grid.Column="6" HorizontalAlignment="Left" VerticalAlignment="Top" Width="92" Height="25" FontSize="10" FontWeight="Bold" FontFamily="Hobo Std" HorizontalContentAlignment="Center" VerticalContentAlignment="Center"/>
```

Comment:
Instead of naming each element to use the name later, use WPF Binding to set the content of controls.
A Control = Button, Label, TextBox etc